### ISSUE TITLE
ngrok latest 64bit version not working + root password

### ISSUE THREAD

**Issue by gnatale** [suda/dvsync#3]
Hi, 

thanks for sharing this code. It was very useful for me although I had two small problems to fix before making it work: 

1) the latest ngrok linux version does not work with the alpine container. Calling "ngrok" within the container returns "file not found" altough the file is in the PATH. I used the 32bits version instead and everything went fine;

2) I had to create a root password on the server otherwise the ssh authentication did not work using the ssh-keys. Not sure why, but after this change the client managed to authenticate and start rsync the data. 

Note that I built the images locally, I did not pull the images on the remote repository.

Hope it helps. Thanks!

**Comment by suda** [comment]
Thanks for this report! Looks like the latest `ngrok` is compiled dynamically against `glibc` which isn't available on Alpine (they use `musl`):

```shell
$ file /usr/local/bin/ngrok
/usr/local/bin/ngrok: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, Go BuildID=DVP4a3w9T55-WTEfnbJf/9FPu4jK0Ygh_2GHoa95_/EKF8V5wc32qtpqApHSrx/dZzLfSrCSIp4rcnSb4qR, not stripped

$ ldd /usr/local/bin/ngrok
	/lib64/ld-linux-x86-64.so.2 (0x7f3c5e51f000)
	libpthread.so.0 => /lib64/ld-linux-x86-64.so.2 (0x7f3c5e51f000)
	libc.so.6 => /lib64/ld-linux-x86-64.so.2 (0x7f3c5e51f000)
Error relocating /usr/local/bin/ngrok: __vfprintf_chk: symbol not found
Error relocating /usr/local/bin/ngrok: __fprintf_chk: symbol not found
```

The 32bit version is statically linked and therefore works:

```
$ file ngrok
ngrok: ELF 32-bit LSB executable, Intel 80386, version 1 (SYSV), statically linked, Go BuildID=x8ohCh31ACQSZJdZ62Cr/bMeMPVlloutMGSnHrbgO/5C_T7SfFWnRz5EEhQhiJ/K9C6onF5BRrNt0QJaBPE, not stripped
```

I'll reach out to ngrok, maybe the dynamic linking was a bug.

**Comment by suda** [comment]
@gnatale I got back from ngrok and there was an unintentional change to their build process causing newer 64-bit binaries to be build dynamically. They'll fix it and the next version will be static but until now I locked `ngrok` version to 2.3.17. Can you confirm this helped?

**Comment by gnatale** [comment]
Hi @suda, sorry for the big delay. I managed to try again only now. 

Yes, now it works well with the 64-bit version. :) 

However, I still had the problem with the dvsync-server root password. These are the steps I had to follow to make it work:

Server side:
- export NGROK_AUTHTOKEN
- build the server image and run the dvsync-server container;
- open a shell on the dvsync-server container and create a password for root:
-- docker-compose exec dvsync-server sh
-- passwd 
- copy DVSYNC_TOKEN

Client side:
- export DVSYNC_TOKEN 
- build client image and run dvsync-client container. 

If I don´t create the root password on the server, rsync fails to connect. 

Hope this helps
